Add reset-password and hash-password commands (#2504)
sensu/sensu-docs: content/sensu-go/5.21/sensuctl/reference.md

sensuctl user change-password --interactive
{{< /highlight >}}

You must specify the user's current password to use the `sensuctl user change-password` command.

#### Reset a user password

To reset a user password without specifying the current password, run:

{{< highlight shell >}}
sensuctl user reset-password USERNAME --interactive
{{< /highlight >}}

You must have admin permissions to use the `sensuctl user reset-password` command.

#### Generate a password hash

> To generate a password hash for the given cleartext password, run:

Author: ```suggestion
To generate a password hash for a specified cleartext password, run:
```

{{< highlight shell >}}

> sensuctl user hash-password

Author: ```suggestion
sensuctl user hash-password PASSWORD
```

{{< /highlight >}}

> The `sensuctl user hash-password` command creates a [bcrypt hash][54] of the password.

Author: ```suggestion
The `sensuctl user hash-password` command creates a [bcrypt hash][54] of the specified password.
```

> The `sensuctl user hash-password` command adds a field to the `User` resource, `password_hash`, whose value is a hash for the user's password.

Reviewer: The `sensuctl user hash-password` command creates a bcrypt hash of a given password.

(It does not add a field to the User resource, although one was added for this release)

> The `sensuctl user hash-password` command adds a field to the `User` resource, `password_hash`, whose value is a hash for the user's password.

Author: ```suggestion
The `sensuctl user hash-password` command creates a [bcrypt hash][54] of the password.
```

[48]: #sensuctl-prune-resource-types
[49]: #examples
[50]: ../../reference/webconfig/
[51]: #update-resources
[52]: #export-resources

> [53]: ../../reference/rbac/#user-specification

Author: ```suggestion
[53]: ../../reference/rbac/#user-specification
[54]: https://en.wikipedia.org/wiki/Bcrypt
```
